src/mame/skeleton/dash4.cpp

```diff
@@ -0,0 +1,85 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/***********************************************************************************************************
+Skeleton driver for Cross Products Ltd. DASH4 Debug Adapter.
+
+DASH4 is the hardware debug adapter of CodeScape development environment for the Hitachi SuperH family of
+microprocessors.
+It's an intelligent debug pod that offers a complete low-level debug capability. The DASH4 uses the JTAG
+interface to communicate with the target at 1.5 Mbytes/s via the Hitachi User Debug Interface (HUDI)
+built-in to the SH7750. The DASH4 connects to the host computer via Ethernet to allow high-speed
```

Comment:
`SH7750` doesn't make sense here: the SoC mounted on the provided PCB ASCII below is a HD6417604F28, which is a SH2, cfr. https://github.com/mamedev/mame/blob/master/src/mame/psikyo/psikyosh.cpp and https://github.com/mamedev/mame/blob/master/src/mame/seibu/feversoc.cpp 
Is the `DASH4` printed on PCB?

Reply:
> Is the DASH4 printed on PCB?
Yes, see attached pic.

This text was based on the (archived) product web page...

![DASH4 PCB front](https://github.com/user-attachments/assets/7af5c8c4-2e7b-44e3-9e04-6cf38486c4fa)


![DASH4 Codescape information](https://github.com/user-attachments/assets/9c575c90-8892-40bf-ac74-089a3a1c81d7)

Reply:
The 7750 is not the CPU on the DASH4, but the CPU externally connected to the DASH4

Reply:
I'm wondering how much similar it is to Katana (Dreamcast) Dev.box's Debug Adapter PCB? expected to be quite similar device but with SCSI instead of Ethernet, uses CodeScape software as well.
![SegaDC_KatanaMBb_1](https://github.com/user-attachments/assets/04ecb429-04e5-4511-ba1d-7369eb0bf7dc)